cli/command/service/update.go

```diff
@@ -269,6 +269,19 @@ func updateService(ctx context.Context, apiClient client.NetworkAPIClient, flags
 		}
 	}
 
+	updateIsolation := func(flag string, field *container.Isolation) error {
+		if flags.Changed(flag) {
+			o := isolationOpts{}
+			val := flags.Lookup(flag).Value.String()
+			err := o.Set(val)
+			if err != nil {
+				return err
+			}
```

Comment:
You don't need to create a new `isolationOpts` struct here. `Value` is that struct, you just have to cast it (like the functions above).